examples/update-demo/README.md

```diff
@@ -92,10 +92,10 @@ We will now update the docker image to serve a different image by doing a rollin
 ```bash
 $ ./4-rolling-update.sh
 ```
-The rollingUpdate command in kubecfg will do 2 things:
+The rollingUpdate command in kubectl will do 2 things:
 
-1. Update the template in the replication controller to the new image (`$DOCKER_HUB_USER/update-demo:kitten`)
-2. Kill each of the pods one by one.  It'll let the replication controller create new pods to replace those that were killed.
+1. Create a new replication controller with a pod template that uses the new image (`$DOCKER_HUB_USER/update-demo:kitten`)
+2. Resize the old and new replication controllers until the new controller replaces the old. This will kill the current pods one at a time, spinnning up new ones to replace them.
 
 Watch the UX, it will update one pod every 10 seconds until all of the pods have the new image.
```

Comment:
I know you didn't change this, but this reads "Watch the User Experience," which doesn't make a terrible amount of sense to me. Is this trying to say, "watch the web page?"